docs/spec/spec.md

```diff
@@ -16,37 +16,49 @@ This document details our _Spec_ and _Status_ customizations.
 
 ## kind: Trigger
 
-### group: eventing.knative.dev/v1beta1
+### group: eventing.knative.dev/v1
 
-_A Trigger represents a subscriber of events with a filter for a specific
-broker._
+_A `Trigger` represents a request to have events delivered to a subscriber from
+a broker's event pool._
 
 ### Object Schema
 
+#### Metadata
+
+Standard Kubernetes
+[meta.v1/ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#objectmeta-v1-meta)
+resource.
+
 #### Spec
 
-| Field        | Type                 | Description                                                                                                                                                                         | Constraints |
-| ------------ | -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
-| broker       | String               | Broker is the broker that this trigger receives events from. Defaults to 'default'.                                                                                                 |             |
-| filter       | TriggerFilter        | Optional Filter is the filter to apply against all events from the Broker. Only events that pass this filter will be sent to the Subscriber. Defaults to subscribing to all events. |             |
-| subscriber\* | pkg/duck.Destination | Subscriber is the addressable that receives events from the Broker that pass the Filter.                                                                                            |             |
+The `TriggerSpec` defines the desired state for the `Trigger`.
 
-\*: Required
+| Field Name   | Field Type                                  | Requirement | Description                                                                                                               | Default Value           |
+| ------------ | ------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------- | ----------------------- |
+| `broker`     | `string`                                    | Required    | The broker that this trigger receives events from.                                                                        | 'default'               |
```

Comment:
I am seeing that sometimes trigger and `Trigger` are used in different places, is this expected? What's the rule to choose between them?

Reply:
My intention was to reduce the unnecessary capitalization for all the objects described in this document, since every use seemed to be written with capitals. The rules I set for my refactoring was that `Trigger` (written as code and with a uppercase first letter) refers to the class of objects (i.e. the CRD for `kind: Trigger`). For all other generic references to triggers I used the lowercase plain word.

Reply:
Thanks for clarification.